src/js/structs/PodContainer.js

```diff
@@ -16,21 +17,27 @@ module.exports = class PodContainer extends Item {
           return PodContainerStatus.RUNNING;
         }
 
-      case 'ERROR':
+      case PodContainerState.ERROR:
         return PodContainerStatus.ERROR;
 
-      case 'FAILED':
+      case PodContainerState.FAILED:
         return PodContainerStatus.FAILED;
 
-      case 'FINISHED':
+      case PodContainerState.FINISHED:
         return PodContainerStatus.FINISHED;
 
-      case 'KILLED':
+      case PodContainerState.KILLED:
         return PodContainerStatus.KILLED;
 
       default:
+        let statusString = this.get('status') || '';
+        if (statusString.startsWith('TASK_')) {
+          statusString = statusString.substr(5);
+        }
+        statusString = statusString.replace(/_/g, ' ');
```

Comment:
I'm not to sure about this "default"/fallback behavior – seems a little to complex. Let's use something like "NA" in case we receive an unknown status.